blog/_src/posts/2016-11-02-beta-reduction-part-1.md

```diff
@@ -0,0 +1,184 @@
+    Title: Beta Reduction (Part 1)
+    Date: 2016-11-02T21:10:18
+    Tags: lambda,calculus,beta,reduction,semantics,by Milo Davis
+
+This post came about because of a research project I became involved with around a year and a half ago in the PRL.  At the time, I had just completed my freshman year and the field of programming languages was completely new to me.  The project involved proving confluence for a restricted subset of β-reduction.  The concepts I had to work with weren't ones that I was familiar with, so I looked online for an approachable explanation, but none existed.  This series of posts is my attempt to rectify this.  This initial post introduces the λ-calculus and β-reduction.  In a follow up post, I will discuss the proofs.
```

Comment:
ACTUALLY I would love to have an even longer introduction (below the `<!-- more -->`). Something to attract the target audience -- if you're about to open Barendregt or you're trying to understand what your cs2500 professor was shouting about during the last week of class.